docs/UserGuide.md

```diff
@@ -424,104 +391,113 @@ Format: **`find`** `FIELD_PREFIX/KEYWORD [FIELD_PREFIX/KEYWORD]…​`
 
-You can export your contact book and share it with your friends.
+You can export your contacts list and share it with your friends with the `export` command.
 
 Format: **`export`** `FILE_NAME.json`
 
-* File can only be exported to JSON format.
-* File will be stored in the `./data` directory in relative to the directory of `Socius.jar`.
+* The exported file follows the JSON format.
+* The exported file will be stored in the `./data` directory from the directory where `Socius.jar` is located.
 
 Examples:
 
-* **`export`** `amy.json` export your contact book details into a file named `amy.json`.
+* **`export`** `friends.json` export your contacts list into a file named `friends.json`.
 
 ### Aliasing commands: `alias`
-
-You can create command shortcut by aliasing it with custom `KEYWORD`. This will assign the `KEYWORD` to a particular `COMMAND`,
-so next time you can simply type in `KEYWORD` instead of the full `COMMAND`.
+You can create shortcuts by aliasing a command with a custom keyword with the `alias` command. 
+This enables you to type the keyword in place of the command.
 
 Format: **`alias`** `a/KEYWORD c/COMMAND`.
 
-* `KEYWORD` cannot be same as command keywords such as `add`.
-* The order of `KEYWORD` and `COMMAND` does not matter.
+* Keywords that are reserved for command names cannot be used as `KEYWORD` (e.g `a/add`).
+* The order of `a/KEYWORD` and `c/COMMAND` does not matter.
 
 Examples:
 
-* **`alias`** `a/Singaporeans c/find nat/Singaporean tg/T08` map the keyword `Singaporeans` to the stated command.
+* **`alias`** `a/Singaporeans c/find nat/Singaporean tg/T08` maps the keyword `Singaporeans` 
+to the `Find` command to find all Singaporeans in tutorial group T08.
 ![result for 'alias a/Singaporeans c/find nat/Singaporean tg/T08'](images/AliasSingaporeans.png)
 *After execution of Alias Command: **`alias`** `a/Singaporeans c/find nat/Singaporean tg/T08`*
 
-### Exiting the program: `exit`
-
-Exits the program.
+### Exiting Socius: `exit`
 
+Exits from Socius.
+f
 Format: **`exit`**
 
 ### Saving the data
 
-Socius data are automatically saved to the computer after any command that changes the data.
-Therefore, there is no need to save manually. The data will be automatically loaded back into the application the next time you use it.
+The data in Socius is automatically saved to your computer after the execution of any command that changes the data.
+There is no need to save manually. The data will be automatically loaded back into Socius on your next use.
 
 ### Command History
 
-Use up/down arrow keys to navigate through command history. This is useful if the command you are going to type only differs slightly from a command you have previously typed, or if you would like to revisit your previous commands.
+Use the Up or Down arrow keys to navigate through command history. This feature is useful if your command is similar to a previous command, or if you wish to revisit your previous commands.
```

Comment:
Minor inconsistency: Perhaps make `Up` and `Down` in code format

Why is it `or` intead of `and`?

Reply:
Fixed both.
And sounds good since user will revisit previous commands either ways